vm_supervisor/network/hostnetwork.py

```diff
@@ -1,26 +1,144 @@
 import logging
+from ipaddress import IPv6Network
 from pathlib import Path
+from typing import Optional, Protocol
+
+from aleph_message.models import ItemHash
+
+from vm_supervisor.conf import IPv6AllocationPolicy
 
 from .firewall import initialize_nftables, setup_nftables_for_vm, teardown_nftables
 from .interfaces import TapInterface
 from .ipaddresses import IPv4NetworkWithInterfaces
+from .ndp_proxy import NdpProxy
 
 logger = logging.getLogger(__name__)
 
 
+def _read_proc_file_value(config_file: Path) -> int:
```

Comment:
This does not seem to be specific to proc files
```suggestion
def _read_file_as_int(config_file: Path) -> int:
```